docs/src/main/asciidoc/_javaee-guide/Jakarta_Mail_API.adoc

```diff
@@ -0,0 +1,4 @@
+[[Jakarta_Mail_API]]
+= Jakarta Mail API
```

Comment:
Could you use `Jakarta Mail` instead of `Jakarta Mail API`?  It is more widely used without "API".